pkg/capabilities/pb/capabilities_helpers.go

```diff
@@ -69,9 +69,14 @@ func CapabilityResponseToProto(resp capabilities.CapabilityResponse) *Capability
 		errStr = resp.Err.Error()
 	}
 
+	var val values.Value
+	if resp.Value != nil {
```

Comment:
This doesn't do anything - if resp.Value is nil then val still remains nil. it looks like you could get rid of "val" althogether?

Reply:
I've had to re-add a variant of this in values.Map.Proto() because the method was panicking when serializing an uninitialized *values.Map pointer.